src/ds/d500/d500-auto-calibration-handler.h

```diff
@@ -0,0 +1,73 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "d500-private.h"
+#include "../../hw-monitor.h"
```

Comment:
Can we include
`#include <hw-monitor.h>`?
or src/hw-monitor?

Reply:
ok